src/validation.js

```diff
@@ -0,0 +1,38 @@
+/* eslint-disable max-len */
+const validateParams = (textToConvert, toCase) => {
+  const cases = ['SNAKE', 'KEBAB', 'CAMEL', 'PASCAL', 'UPPER'];
```

Comment:
Better way use CASES_TO_CONVERT ( or something like that) for constant name.
Also, move all constants (status code, error messages) to the upper scope or better way - to separate file.

Reply:
Hi! 
Fixed everything, thanks!